gax-java/gax/src/test/java/com/google/api/gax/rpc/ApiClientHeaderProviderTest.java

```diff
@@ -40,6 +40,7 @@ public class ApiClientHeaderProviderTest {
 
   private static final String X_GOOG_API_CLIENT = "x-goog-api-client";
   private static final String CLOUD_RESOURCE_PREFIX = "google-cloud-resource-prefix";
+  private static final String API_VERSION_HEADER_KEY = "x-goog-api-version";
 
   @Test
   public void testServiceHeaderDefault() {
```

Comment:
Can you add an assertion to the default test case to show that `provider.getHeaders()` does not have `API_VERSION_HEADER_KEY`